packages/simcore-sdk/tests/unit/test_node_ports_common_file_io_utils.py

```diff
@@ -1,31 +1,82 @@
 # pylint: disable=redefined-outer-name
 # pylint: disable=unused-argument
 
-from asyncio import BaseEventLoop
+from typing import AsyncIterable, Iterable
 
 import pytest
-from aiohttp import web
-from aiohttp.test_utils import TestClient
+from aiohttp import ClientResponse, ClientSession
+from aioresponses import aioresponses
 from simcore_sdk.node_ports_common.file_io_utils import (
     ExtendedClientResponseError,
+    _check_for_aws_http_errors,
     _raise_for_status,
 )
 
-TEST_ERROR = "OPSIE there was an error here"
+A_TEST_ROUTE = "http://127.0.0.1:1249/test-route"
 
 
 @pytest.fixture
-def test_client(loop: BaseEventLoop, aiohttp_client) -> TestClient:
-    async def raise_error(request):
-        return web.Response(body=TEST_ERROR, status=web.HTTPBadRequest.status_code)
+def mock_service() -> Iterable[aioresponses]:
+    with aioresponses() as mock:
+        yield mock
 
-    app = web.Application()
-    app.router.add_get("/", raise_error)
-    return loop.run_until_complete(aiohttp_client(app))
 
+@pytest.fixture
+async def aiohttp_client() -> AsyncIterable[ClientSession]:
```

Comment:
Question, is that not already a fixture from pytest-aiohttp????
why not directly use it?

Reply:
Made a mess with the names. I was looking for something else. I required an `aiohttp.ClientSession`